### PR TITLE
Fix Memoizable test

### PR DESCRIPTION
**Pull request by dkubb**

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb, @snusnu I see no value in the 2.3 and 2.4 runs, lets kill them. If people want to use older ruby, they can use older mutant. so we can keep AUOM corpus.

File: lib/mutant/matcher/method/instance.rb [hunk 1]
```diff
@@ -12,10 +12,12 @@ class Instance < self
         # @param [UnboundMethod] method
         #
         # @return [Matcher::Method::Instance]
+        #
+        # :reek:ManualDispatch
         def self.new(scope, target_method)
           name = target_method.name
           evaluator =
-            if scope.include?(Memoizable) && scope.memoized?(name)
+            if scope.respond_to?(:memoized?) && scope.memoized?(name)
```
Comment (mbj) [inline lib/mutant/matcher/method/instance.rb]: Given mutant has a wide user base, and `memoized?` is a rellative common method name, should we test for both `include?(Memoizable) && respond_to?(:memoized?)``

Comment (dkubb) [reply lib/mutant/matcher/method/instance.rb]: @mbj I wasn't able to kill that without creating a completely synthetic test case.

Comment (mbj) [reply lib/mutant/matcher/method/instance.rb]: @dkubb I'm fine with a synthetic test case. This class has verbose unit test coverage, adding this case should be possible. I'm fine to do such a fixup to this commit.

File: Changelog.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# v0.8.19 2018-11-06
+
+* Fix bug when mutating a module that includes Memoizable
```
Comment (mbj) [inline Changelog.md]: Lets also mention we dropped Ruby < 2.5 support.

Comment (dkubb) [reply Changelog.md]: @mbj Done.

**Pull request merged**